lib/services/http_service.rb

```diff
@@ -58,6 +72,10 @@ def make_request(type = :get, endpoint = '/', query_params = nil, body = nil)
         request[CONTENT_TYPE] = APPLICATION_JSON
       end
 
+      if headers
+        headers.each { |key,value| request[key] = value }
```

Comment:
headers.each { |key,value| request[key] = value } if headers.present?  is a more concise, ruby like way to do this.  if the line is long, i generally dont use suffix expressions. if short, i do .  this is in the middle in think :)  so, whatever.